README.md

```diff
@@ -40,13 +41,32 @@ Starter code is already provided in `controllers/main.controllers.js` file.<br>
 ## How to Contribute
 - Create the issue.
 - Fork the Repo.
+- Create a new Branch [git checkout -b "new branch name"]
 - Fix the issue.
 - Generate a Pull Request.
 - Wait for your PR to get merged.
 
+
+## Documentation
```

Comment:
Remove this section documentation should explain about the current repo. 
This blog gives generalized idea that is not required.